lib/src/entities/authenticator_impl.dart

```diff
@@ -70,10 +70,9 @@ class AuthenticatorImpl with WidgetsBindingObserver implements Authenticator {
         final lastActive = await _repository.getPausedTimestamp();
         if (lastActive != null) {
           final now = DateTime.now();
-          if (now.millisecondsSinceEpoch - lastActive.millisecondsSinceEpoch > lockAfterDuration.inMilliseconds) {
-            _lockController.lock(
-              availableMethods: await getAvailableBiometricMethods(),
-            );
+          if (now.millisecondsSinceEpoch - lastActive.millisecondsSinceEpoch >
+              lockAfterDuration.inMilliseconds) {
+            _lockWithBiometricMethods();
```

Comment:
I think the logic here should be the same with https://github.com/DutchCodingCompany/pin_lock/pull/14/files#diff-5c49c804fe2bc40c4001e93d44b185e624e5db9b53a68298ff3390eddc93b1d4R328 so I extracted it into a method called `_lockWithBiometricMethods()`